spec/ship_spec.rb

```diff
@@ -1 +1,39 @@
-#Created ship_spec.rb file
+require 'rspec'
+require './lib/ship'
+
+RSpec.describe Card do
+# Setup for all tests (create a Ship)
+  before(:each) do 
+    @cruiser = Ship.new("Cruiser", 3)
+  end
+  describe "#initialize" do 
+    it "is an instance of Ship" do
+      expect(@cruiser).to be_an_instance_of(Ship)
+    end
+
+    it "has a readable name, length, and health" do 
+      expect(@cruiser.name).to eq "Cruiser"
+      expect(@cruiser.legnth).to eq 3
```

Comment:
spelling correction for @cruiser.legnth (change to @cruiser.length)

Reply:
Edit complete